app/app.go

```diff
@@ -394,6 +394,17 @@ func (a *App) GetReviewApps() ([]*ReviewApp, error) {
 	return reviewApps, nil
 }
 
+func (a *App) ReviewAppExists() (bool, error) {
+	if !a.Pipeline {
+		return false, fmt.Errorf("%s is not a pipeline and cannot have review apps", a.Name)
+	}
+	_, err := SsmParameter(a.Session, fmt.Sprintf("/apppack/pipelines/%s/review-apps/pr/%s", a.Name, *a.ReviewApp))
```

Comment:
You might actually need to check the value of this. It should be JSON with a `status` key that needs a value of `created`. See https://github.com/apppackio/apppack/blob/main/cmd/reviewapps.go#L46

Reply:
The parameter gets created when the PR is created at GitHub. That functionality is in the builder here, https://github.com/apppackio/apppack-codebuild-image/blob/main/builder/build/prebuild.go#L301-L304

In that case, the parameter will exist, but the review app will not.